Doc/library/sqlite3.rst

```diff
@@ -1224,6 +1252,38 @@ Connection objects
 
       .. versionadded:: 3.11
 
+   .. attribute:: autocommit
+
+      This attribute controls :pep:`249`-compliant transaction behaviour.
+      :attr:`!autocommit` has three allowed values:
+
+      * ``False``: Select :pep:`249`-compliant transaction behaviour,
+        implying that :mod:`!sqlite3` ensures a transaction is always open.
+        Use :meth:`commit` and :meth:`rollback` to close transactions.
+
+        This is the recommended value of :attr:`!autocommit`.
+
+      * ``True``: Use SQLite's `autocommit mode`_.
```

Comment:
```suggestion
      * ``True``: Use SQLite's `autocommit mode`_,
      implying that :mod:`!sqlite3` does not implicitly open any transactions at all.
```

Reply:
We deliberately keep the reference succinct. Please see previous discussions in this PR. For further explanation, we link to the _Explanation_ section.

Reply:
I added this clause because in the same list you used a similar clause for `autocommit=False`:

>  * ``False``: Select :pep:`249`-compliant transaction behaviour,
>          implying that :mod:`!sqlite3` ensures a transaction is always open.

Reply:
If this is a problem we can address it in a follow-up docs fixup.